ClinGen/clincoded
# Connects to #1523. Added gnomAD data (from myvariant.info) to Population tab of VCI

Steps to test #1523:
1. From the dashboard, click on the "Select Variant for Variant Curation" link.
2. Go through the process of submitting a ClinVar or CA ID.
3. Once the evidence view is rendered, select the "Population" tab.
4. Confirm the expected results in the gnomAD table (and possibly the HMAF section, which was updated to consider gnomAD data).

A couple of sample IDs:
12345 (ClinVar) - no population data available
13346 (ClinVar) - not an SNV (so no data displayed)
17000 (ClinVar) - has gnomAD data
21134 (ClinVar) - has gnomAD data (which has the HMAF)
CA501097 (CA) - no population data available
